data_models/sentence_with_attributes.py

```diff
@@ -0,0 +1,27 @@
+''' Class definition for SentenceWithAttributes object'''
+
+
+class SentenceWithAttributes:
```

Comment:
The main motivation to do so was the fact that there were tests which were using these modules and it would be preferable for them to be placed in data_models/

Reply:
What are these tests for? Could you elaborate a bit more?

Reply:
as an example we use the sentence_with_attributes object in the text media matching tests : 
https://github.com/googleinterns/stampify/blob/2a834db30a96e8254547dc5ee7a42c8fb970c3ff/tests/summarizer/text_media_input_fetcher.py#L10

Reply:
on second thought moved it to the summarizer/